### PR TITLE
Fit for MMPretrain EfficientFormer

### PR DESCRIPTION
**Pull request by huayuan4396**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: tests/regression/mmpretrain.yml [hunk 1]
```diff
@@ -237,3 +237,11 @@ models:
     pipelines:
```
Comment (RunningLeon) [inline tests/regression/mmpretrain.yml:237]: pls. update the docs for supported models in benchmark.md mmpretrain.md, etc.

File: docs/en/04-supported-codebases/mmpretrain.md [hunk 2]
```diff
@@ -185,3 +185,4 @@ Besides python API, mmdeploy SDK also provides other FFI (Foreign Function Inter
 | [MobileOne](https://github.com/open-mmlab/mmpretrain/tree/main/configs/mobileone)                  |      Y      |      Y       |    N     |  N   |   ?   |    N     |
 | [EfficientNet](https://github.com/open-mmlab/mmpretrain/tree/main/configs/efficientnet)            |      Y      |      Y       |    N     |  N   |   ?   |    N     |
 | [Conformer](https://github.com/open-mmlab/mmpretrain/tree/main/configs/conformer)                  |      Y      |      Y       |    N     |  N   |   ?   |    N     |
+| [Conformer](https://github.com/open-mmlab/mmpretrain/tree/main/configs/efficientformer)            |      Y      |      Y       |    Y     |  N   |   ?   |    N     |
```
Comment (RunningLeon) [inline docs/en/04-supported-codebases/mmpretrain.md:188]: Conformer -> EfficientFormer

File: docs/en/03-benchmark/benchmark.md [hunk 3]
```diff
@@ -625,6 +625,28 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
     <td align="center">-</td>
     <td align="center">-</td>
     <td align="center">-</td>
+  <tr>
+    <td align="center" rowspan="2"><a href="https://github.com/open-mmlab/mmpretrain/blob/main/configs/efficientformer/efficientformer-l1_8xb128_in1k.py">EfficientFormerr</a></td>
```
Comment (RunningLeon) [inline docs/en/03-benchmark/benchmark.md:629]: EfficientFormerr -> EfficientFormer

File: docs/zh_cn/03-benchmark/benchmark.md [hunk 4]
```diff
@@ -618,6 +619,30 @@ GPU: ncnn, TensorRT, PPLNN
     <td align="center">95.61</td>
     <td align="center">-</td>
     <td align="center">-</td>
+    <td align="center">-</td>
+  <tr>
+    <td align="center" rowspan="2"><a href="https://github.com/open-mmlab/mmpretrain/blob/main/configs/efficientformer/efficientformer-l1_8xb128_in1k.py">EfficientFormerr</a></td>
```
Comment (RunningLeon) [inline docs/zh_cn/03-benchmark/benchmark.md:624]: EfficientFormerr -> EfficientFormer

File: docs/zh_cn/04-supported-codebases/mmpretrain.md [hunk 5]
```diff
@@ -190,3 +190,4 @@ for label_id, score in result:
 | [MobileOne](https://github.com/open-mmlab/mmpretrain/tree/main/configs/mobileone)                  |      Y      |      Y       |    N     |  N   |   ?   |    N     |
 | [EfficientNet](https://github.com/open-mmlab/mmpretrain/tree/main/configs/efficientnet)            |      Y      |      Y       |    N     |  N   |   ?   |    N     |
 | [Conformer](https://github.com/open-mmlab/mmpretrain/tree/main/configs/conformer)                  |      Y      |      Y       |    N     |  N   |   ?   |    N     |
+| [Conformer](https://github.com/open-mmlab/mmpretrain/tree/main/configs/efficientformer)            |      Y      |      Y       |    Y     |  N   |   ?   |    N     |
```
Comment (RunningLeon) [inline docs/zh_cn/04-supported-codebases/mmpretrain.md:193]: Conformer -> EfficientFormer

Comment (RunningLeon) [reply docs/zh_cn/04-supported-codebases/mmpretrain.md:193]: are ncnn and openvino not supported?

Comment (huayuan4396) [reply docs/zh_cn/04-supported-codebases/mmpretrain.md]: > are ncnn and openvino not supported?

I re-test it, openvino is supported but the convert result of ncnn is False.

**Review by AllentDan — APPROVED** [review]

File: tests/regression/mmpretrain.yml [hunk 6]
```diff
@@ -237,3 +237,12 @@ models:
...
       - *pipeline_trt_dynamic_fp32
+
+  - name: EfficientFormer
+    metafile: configs/efficientformer/metafile.yml
+    model_configs:
+      - configs/efficientformer/efficientformer-l1_8xb128_in1k.py
+    pipelines:
+      - *pipeline_ts_fp32
+      - *pipeline_ort_dynamic_fp32
+      - *pipeline_trt_dynamic_fp32
```
Comment (RunningLeon) [inline tests/regression/mmpretrain.yml:248]: may add openvino pipline here

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**